src/System.Diagnostics.DiagnosticSource/src/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      net45-Windows_NT;
```

Comment:
I think @tarekgh was hitting this while trying to make netfx build configurations work so I suspect he will take care of removing this. However in general we should avoid adding configurations that don't work yet unless we have follow-up items to make them work. 

For this library what is the goal of adding the net45 configuration?